app/controllers/weeks_controller.rb

```diff
@@ -6,4 +6,12 @@ def destroy
     Week.find(params[:id]).destroy
     render plain: '', status: :ok
   end
+
+  def delete_multiple
```

Comment:
Instead of `delete_multiple` that operates on a set of week ids, I think this should operate at the level of the course, taking a course_id param and doing something like course.weeks.destroy_all. That way, there's less danger of leaving the data in an inconsistent state, if the browser doesn't have the up-to-date set of week ids (which may happen if another user was editing the timeline in the meantime). The addWeek action to create a week on the client-side sets a placeholder ID using Date.now(), so it's also possible that there would be a week where the id differs from what is in the database.

Before the action, it should also verify that the current user has permission. (I notice that we don't do this with the existing `destroy` method, which we should do.)